lib/api/controller/realtime.js

```diff
@@ -107,7 +107,7 @@ class RealtimeController extends NativeController {
 
     this.kuzzle.tokenManager.unlink(request.context.token, roomId);
 
-    return { roomId };
+    return roomId;
```

Comment:
I don't get it: the previous version seems to match the documented returned value (https://docs.kuzzle.io/core/2/api/controllers/realtime/unsubscribe/#response)

Reply:
with `{ roomId }` as returned value it's not the response will not have the `result` level

Reply:
As we discussed, you did a correction, not a mistake, I'm closing this pr